### PR TITLE
Updating MAINTAINERS.md with UI project maintainers details

### PR DESCRIPTION
**Pull request by vishnoianil**

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -168,6 +168,7 @@ Team which has full maintainer access to the Taxonomy repository
 - [`kelbrown20`](https://github.com/kelbrown20)
 - [`luke-inglis`](https://github.com/luke-inglis)
 - [`mingxzhao`](https://github.com/mingxzhao)
+- [`mrutkows`](https://github.com/mrutkows)
```
Comment (bjhargrave) [inline MAINTAINERS.md:171]: @russellb Is this still temporary?

https://github.com/instructlab/community/pull/323#issuecomment-2177365699

Comment (bjhargrave) [reply MAINTAINERS.md:171]: @vishnoianil If you remove this line and amend your commit, I can approve.

Comment (jjasghar) [reply MAINTAINERS.md:171]: Yep, waiting on @mrutkows to tell me when his demo is completed.

Comment (vishnoianil) [reply MAINTAINERS.md:171]: @bjhargrave This was generated through the maintainers.py tool, so i didn't manually update it. Let me fix it and push the change.

Comment (vishnoianil) [reply MAINTAINERS.md:170]: Fixed it.

Comment (bjhargrave) [reply MAINTAINERS.md:170]: You removed the wrong line. Matt stays on line 36 but not 170.

**Review by bjhargrave — APPROVED** [review]

File: MAINTAINERS.md (hunk 1 above)
Comment (vishnoianil) [reply MAINTAINERS.md]: yup, realized it after the push :-) . fixed it.

**Pull request merged**